packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/example-print-extension.doc.ts

```diff
@@ -0,0 +1,281 @@
+          title: 'Page breaks',
+          sectionContent: `When returning multiple documents, use CSS page breaks to ensure proper printing:
+
+\`\`\`css
+@media print {
+  .page-break {
+    page-break-after: always;
+  }
+}
+\`\`\``,
+        },
+        {
+          type: 'info',
+          title: 'Email obfuscation',
+          sectionContent: `When using Cloudflare tunnels for development, wrap email addresses in HTML comments to prevent obfuscation:
+
+\`<!--email_off-->example@email.com<!--/email_off-->\`
+
+[Learn more about email obfuscation](https://developers.cloudflare.com/waf/tools/scrape-shield/email-address-obfuscation/#prevent-cloudflare-from-obfuscating-email)`,
+        },
+      ],
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'extension-ui',
+      title: 'Build the extension UI',
+      sectionContent: `
+Your print extension needs two main components:
+
+1. A **Tile** on the POS home screen that launches your extension modal
+2. A **Modal** that handles document preview and printing
+
+Here's how to implement both components:
+      `,
+      codeblock: {
+        title: 'Extension Components',
+        tabs: [
+          {
+            title: 'HomeTile.tsx',
+            code: `${examplePath}/print-tile.tsx`,
+            language: 'tsx',
+          },
+          {
+            title: 'HomeModal.tsx',
+            code: `${examplePath}/print-modal.tsx`,
+            language: 'tsx',
+          },
+        ],
+      },
+      sectionNotice: [
+        {
+          type: 'note',
+          title: 'Key components',
+          sectionContent: `
+- The \`Tile\` component uses \`api.action.present()\` to open the modal
+- The \`PrintPreview\` component displays the document before printing
+- The Print API's \`print()\` method opens the native print dialog
+- Loading states and error handling ensure a smooth printing experience`,
+        },
+      ],
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'extension-config',
+      title: 'Configure your extension',
+      sectionContent: `
+Configure your extension with the necessary permissions and settings in the \`shopify.extension.toml\` file:
+      `,
+      codeblock: {
+        title: 'Extension Configuration',
+        tabs: [
+          {
+            title: 'shopify.extension.toml',
+            code: `${examplePath}/shopify.extension.toml`,
+            language: 'toml',
+          },
+        ],
+      },
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'testing',
+      title: 'Testing your extension',
+      sectionContent: `
+To test your print extension:
+
+1. Start your development server:
+   \`\`\`bash
+   shopify app dev
+   \`\`\`
+
+2. Open the POS app and install your development extension
+3. Navigate to your print extension
+4. Test the preview and print functionality
+
+[Learn more about how to test your extension](/docs/api/pos-ui-extensions/unstable/getting-started#step-2-run-the-local-extension-in-your-development-store)
```

Comment:
Oh, I see that's here. Maybe you could tie this link together with the above numbered list?